.github/workflows/e2e.yaml

```diff
@@ -149,10 +149,64 @@ jobs:
         name: tests-report
         path: istio-${{ matrix.kubernetes-version }}-${{ matrix.istio-version }}-tests.xml
 
+  e2e-gke-tests:
+    environment: "gcloud"
+    runs-on: ubuntu-latest
+    steps:
+    - name: setup golang
+      uses: actions/setup-go@v3
+      with:
+        go-version: '^1.19'
+
+    - name: cache go modules
+      uses: actions/cache@v3
+      with:
+        path: ~/go/pkg/mod
+        key: ${{ runner.os }}-build-codegen-${{ hashFiles('**/go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-build-codegen-
+
+    - name: checkout repository
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - uses: Kong/kong-license@master
+      id: license
+      with:
+        # PULP_PASSWORD secret is set in "Configure ci" environment
+        password: ${{ secrets.PULP_PASSWORD }}
+
+    - name: run ${{ matrix.kubernetes-version }}
+      run: make test.e2e.gke
+      env:
+        GOOGLE_APPLICATION_CREDENTIALS: ${{ secrets.GOOGLE_APPLICATION_CREDENTIALS }}
+        GOOGLE_PROJECT: ${{ secrets.GOOGLE_PROJECT }}
+        GOOGLE_LOCATION: ${{ secrets.GOOGLE_LOCATION }}
+        KUBERNETES_MAJOR_VERSION: 1
+        KUBERNETES_MINOR_VERSION: 24
```

Comment:
I think this configuration will always run GKE with k8s 1.24 because `KUBERNETES_MINOR_VERSION` is always `24` . Also, no `matrix` section is present is this part. So, the appearance of `${{ matrix.kubernetes-version }}` should be changed to maybe `1.24`, OR define a proper `matrix` in this section.